### PR TITLE
feat: GovernanceRouter unit tests in fork mode

### PR DESCRIPTION
**Pull request by odyslam**
## Motivation

Test the GovernanceRouter with on-chain state

## Solution

Refactor (a bit) the unit tests and inherit them to the fork environment

## PR Checklist

- [ ] Added Tests
- [ ] Updated Documentation
- [ ] Updated CHANGELOG.md for the appropriate package

### CODE REVIEW

**Comment by anna-carroll** [comment]
looks good i think. does governance router have any tests that interact with real home functionality?

File: packages/contracts-ops/contracts/test/Reboot.t.sol [hunk 1]
```diff
@@ -27,56 +33,70 @@ contract RebootTest is
             NomadTestWithUpdaterManager,
             ReplicaHandlers,
             ReplicaTest,
-            HomeTest
+            HomeTest,
+            GovernanceRouterTest
         )
     {
+        // ALL
         vm.createSelectFork(vm.envString("RPC_URL"), 15_977_625);
```
Comment (prestwich) [inline packages/contracts-ops/contracts/test/Reboot.t.sol:41]: This function is too large, I can't tell what it's doing. Break it up into discrete steps

Comment (prestwich) [reply packages/contracts-ops/contracts/test/Reboot.t.sol:41]: rule of thumb: anything that invokes writejson should be in a different scope that pairs it with the thing that generated whatever it is that is being written

**Comment by anna-carroll** [comment]
refactored: https://github.com/nomad-xyz/monorepo/pull/544

**Pull request closed without merging**